### DISCUSSION TITLE
Making summary tables of thresholds used in tasks

### DISCUSSION THREAD

**Discussion by ChristopheLegendre** [tgen/phoenix#398] [Ideas]
Hi Contributors and Developers,

I was wondering if it were possible to make a datasheet or a table summarizing the important thresholds used in Phoenix' tasks pipeline.
For instance, we know that `Copy Number` module contains tasks where thresholds are applied, statically or dynamically set, to the data, such as filtering the `Hets` in the normal sample, `minimum depth`, `binning` values, etc.  
Same thing with `vcfMerger2` module where some thresholds are statically assigned, or the `Tx capture` module, `tumour only` module etc.

This would also help the pipeline users to get why some expected results are the way they are.

What do you think?

If we already have one similar table, could you point me to it and also add the link to the Phoenix's README page.

Thanks.
Christophe

**Comment by PedalheadPHX** [comment]
Sounds like a great idea, since you suggested it why don't you start it :)
